demos/demo_solo12_calibration.cpp

```diff
@@ -1,60 +0,0 @@
-/**
```

Comment:
Why removing this demo?

Reply:
I did a mistake merging the demos/CMakeLists.txt BTW there shouldn't be the demo_solo12_calibration as you deleted the file.
I can fix it but I wonder why?